ci/docker-pull-or-build-and-push.sh

```diff
@@ -31,7 +38,7 @@ retry 3 docker-compose --ansi never pull $image_name
 # (since v1.25.1, `docker-compose pull` is NOT failing when it can't fetch the image).
 image_tag=$(docker-compose --ansi never config | yq eval ".services.$image_name.image" -)
 docker image inspect "$image_tag" &>/dev/null || {
-  docker-compose --ansi never build --build-arg user_id="$(id -u)" --build-arg group_id="$(id -g)" $image_name
+  docker-compose --ansi never build --build-arg user_id=$USER_ID --build-arg group_id=$GROUP_ID $image_name
```

Comment:
For clarity: is it possible to just select the user/group with `user:` in docker-compose.yml **without** having the user previously created in the image? I'd intuitively say **no**, but I've seen a solution like that being PoCed on one of your PRs, following certain blog post 🤔

Reply:
Well, i tried it like it was presented in the blog and also tried different approaches but it doesn't work, i always got default root user inside the container.